src/commands/setup.js

```diff
@@ -104,11 +104,11 @@ function setup(region, defaults) {
         (err, info) => {
 
             if(err) {
-                this.logger.error('There has been a problem with toter setup ' 
+                this.logger.error('There has been a problem with toter setup '
```

Comment:
this.logger.error is uses the same function signature as console.error. Hence you can simply do:
```js{4}
this.logger.error('There has been a problem with toter setup', err)
```

Reply:
I didn't actually write this code - but I totally agree with you and this is how I would have written it myself!  Change made.